packages/docusaurus/src/client/templates/index.html.template.ejs

```diff
@@ -5,8 +5,13 @@
     <meta name="viewport" content="width=device-width">
     <meta name="generator" content="Docusaurus">
     <title><%= htmlWebpackPlugin.options.title %></title>
+    <%= htmlWebpackPlugin.options.headTags %>
+    <%= htmlWebpackPlugin.tags.headTags %>
```

Comment:
Is this leftover by mistake?

Reply:
no its not. So html webpack plugin by default will inject the required script at postbody. So since we want to add our own postbody tags, we have to set “inject:false” to htmlwebpackplugin and then put the injection place. See https://github.com/jantimon/html-webpack-plugin/tree/master/examples/custom-insertion-position

Reply:
*at head and postbody.